airbyte-integrations/connectors/source-datadog/source_datadog/manifest.yaml

```diff
@@ -65,7 +65,7 @@ definitions:
     type: "DefaultPaginator"
     pagination_strategy:
       type: "CursorPagination"
-      cursor_value: "{{ last_records['meta', 'page', 'after'] }}"
```

Comment:
do we know if this even worked? I'd expect `last_records` to be an array, not an object